Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:slim as build-env
+
+WORKDIR /home/faucet/
+COPY . .
+
+RUN npm install
+
+# Use Alpine and install Node.js which is 50% smaller than the -alpine version of the node
+# image (53 MB including the faucet app).
+FROM alpine:3.15
+# Installs Node.js 16 (https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.15)
+RUN apk add --update nodejs
+RUN apk add --update ca-certificates jq bash curl
+
+WORKDIR /home/faucet
+
+COPY . .
```

Comment:
why copy the code?
No need to copy the bundle itself?
What is the benefit of running npm build in the above build docker env then? is it just a check?

Reply:
If that is the case, can you check the size of the node:slim image and alpine image with dependencies installed.
Ideally we do this to reduce the size of the final image, but if node:slim itself is ok comparable size, then 2 images are not needed

Reply:
he is doing npm install actually, yea. we should do build. it is better, then have the dist moved

Reply:
npm build is an internal function already called by npm install, but there is nothing to build its a js which can just be run right ?

Reply:
Size with both is around ~150MB and just with slim is ~250MB.